src/resolve/resolved-dns-stub.c

```diff
@@ -949,6 +950,21 @@ static void dns_stub_process_query(Manager *m, DnsStubListenerExtra *l, DnsStrea
                 return;
         }
 
+        if (m->refuse_record_types) {
```

Comment:
no need for the "if" block, STRV_FOREACH will be a no-op already if empty

Reply:
@bluca removed this code.